cassandra-reaper.advisories.yaml

```
@@ -0,0 +1,32 @@
+schema-version: 2.0.2
+
+package:
+  name: cassandra-reaper
+
+advisories:
+  - id: CVE-2015-0886
+    aliases:
+      - GHSA-9h6p-92jq-888x
+    events:
+      - timestamp: 2024-01-11T10:02:13Z
+        type: pending-upstream-fix
+        data:
+          note: To be able to fix this CVE, we need to upgrade the "org.apache.cassandra:cassandra-all" dependency version from `2.2.12` which is the version project is currently using to at least `4.0.10` which is the version we should bump to to fix the CVEs but we can't because the build was failed due to compilation errors.
+
+  - id: CVE-2020-8908
+    aliases:
+      - GHSA-5mg8-w23w-74h3
+    events:
+      - timestamp: 2024-01-11T09:54:49Z
+        type: pending-upstream-fix
+        data:
+          note: Pending upstream fix, this fix will require some code changes since when we upgrade the "com.google.guava:guava" dependency version from `24.1.1-jre` which is the version project is currently using to `32.0.0-android` which is the version we should bump to to fix the CVEs but we can't because the build was failed due to compilation errors.
```

Perhaps we can just mention the version here without the variant?

```suggestion
          note: Pending upstream fix, this fix will require some code changes since when we upgrade the "com.google.guava:guava" dependency version from 24.1.1 which is the version project is currently using to 32.0.0 which is the version we should bump to to fix the CVEs but we can't because the build was failed due to compilation errors.
```